src/dataflow/src/server.rs

```diff
@@ -185,31 +185,41 @@ pub enum Command {
 }
 
 impl Command {
-    /// Produces a copy of the command suitable for the indicated worker.
+    /// Partitions the command among `peers` many workers.
```

Comment:
To be precise, this doesn't need to partition for workers, but more general receivers. In the cluster case, we partition per process, which then partitions per local worker.